charts/prometheus/templates/prometheus-alerts-configmap.yaml

```diff
@@ -1299,7 +1299,7 @@ data:
               10 minutes."
 
         - alert: NginxIngressHigh5XXRate
-          expr: sum by (ingress) (rate(nginx_ingress_controller_requests{status=~"^4.."}[2m])) / sum by (ingress) (rate(nginx_ingress_controller_requests{ingress!=""}[2m])) * 100 > 1
+          expr: sum by (ingress) (rate(nginx_ingress_controller_requests{status=~"^5.."}[2m])) / sum by (ingress) (rate(nginx_ingress_controller_requests{ingress!=""}[2m])) * 100 > 1
```

Comment:
I think this change is correct, but I would like to mention this will be noisy because the "waiting for airflow to start up" is 503.

Reply:
yeah all good, I wanted to at least make this little fix because it was flat wrong before. But not take it anything in this PR because this pr wasn't about that. I agree we should limit the scope of the alert to not look at customer namespaces as a separate PR.